lib/llm/message.rb

```diff
@@ -4,9 +4,14 @@ module LLM
   class Message
     attr_accessor :role, :content
 
-    def initialize(role, content)
+    def initialize(role, content, extra = nil)
```

Comment:
This should default to an empty hash (`extra = {}`)

Reply:
Thanks! I just missed that.